src/dao/ExchangeDAO.js

```diff
@@ -4,8 +4,16 @@ import OtherContractsDAO from './OtherContractsDAO'
 import LHTProxyDAO from './LHTProxyDAO'
 import AssetProxyDAO from './AssetProxyDAO'
 import ExchangeContractModel from '../models/contracts/ExchangeContractModel'
+import web3Provider from '../network/Web3Provider'
+import TransactionModel from '../models/TransactionModel'
+import { Map } from 'immutable'
 
 export class ExchangeDAO extends AbstractOtherContractDAO {
+  events = {
+    SELL: 'Sell',
+    BUY: 'Buy'
```

Comment:
this is the constants, so define them as a constants
to make convenient interface (without need to export extra variables from ExchangeDAO file) just write some new wrapper-methods like it already, for instance, done in DAOFactory.js